account_template_active/readme/USAGE.rst

```diff
@@ -0,0 +1,7 @@
+* Go to Configuration > Templates
+* Select then the template you want to change.
+
+**Note:**
+
+You could need the OCA module ``account_menu`` (same repository) to
```

Comment:
I think this part is obsolete, as you introduce a dependency to ``account_menu`` in the manifest.

Reply:
Ah thanks! Removed.